aip/0133.md

```diff
@@ -161,6 +161,8 @@ message CreateBookRequest {
 - The documentation **should** explain what the acceptable format is, and the
   format **should** follow the guidance for resource name formatting in
   [AIP-122][].
+- If a user tries to create a resource with an ID that would result in a
+  duplicate resource name, the service **must** error with `ALREADY_EXISTS`.
```

Comment:
Hmmm... that could leak information about the existence of the other resource. If this is a resource they wouldn't have access to anyway, it should presumably return forbidden?

Reply:
Fair point. Will clarify.

Reply:
Done. (Now, go enjoy your weekend! ;-))